test_parenthetics.py

```diff
@@ -0,0 +1,34 @@
+import pytest
+from parenthetics import parenthetics
+
+
+def test_single_open():
+    assert parenthetics("(") == 1
+
+
+def test_single_close():
+    assert parenthetics(")") == -1
+
+
+def test_single_balanced():
+    assert parenthetics("()") == 0
+
+
+def test_multi_open_close_1():
+    assert parenthetics("(()") == 1
+
+
+def test_multi_open_close_2():
+    assert parenthetics("())") == -1
+
+
+def test_broken():
+    assert parenthetics("))((") == -1
+
+
+def test_text():
+    assert parenthetics("This is a silly test!") == 0
+
+
+def test_text_parens():
+    assert parenthetics("This is a silly test (but not really that silly)!") == 0
```

Comment:
I'd like to see a couple more test cases that could possibly break your code. You don't test an empty string, and you don't test anything more complex like (()()))(). It might seem silly to test these because you know that your code will breeze through them, but your tests shouldn't assume anything about your code. When you're working with code, it could change completely over time and if you don't have exhaustive test cases, it could break without you realizing.